docs/content/reference/resource-schema/core-schema/environment-schema/index.md

```diff
@@ -24,27 +24,30 @@ weight: 000
 |------|:--------:|-------------|---------|
 | [compute](#compute) | y | Container runtime configuration. | [See below](#compute)
 | [recipeConfig](#recipeconfig) | n | Configuration for Recipes. Defines how each type of Recipe should be configured and run. | [See below](#recipeconfig) 
-| [recipes](#recipes) | n | Recipes registered to the environment | [See below](#recipes)
+| [recipes](#recipes) | n | Recipes registered to the environment. | [See below](#recipes)
 | simulated | n | When enabled, a simulated environment will not deploy any output resources or run any Recipes when an application is deployed. This is useful for dry runs or testing. Defaults to `false`. | `true`
 | [extensions](#extensions) | n | The environment extension. | [See below](#extensions)
+| [providers](#providers) | n | The cloud provider configuration, either `azure` or `aws`. | [See below](#providers)
 
 ### compute
 
 Details on what to run and how to run it are defined in the `container` property:
 
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
-| kind | y | The kind of container runtime to use. Only option is `'kubernetes'` | `'kubernetes'`
-| namespace | y | The Kubernetes namespace to render application resources into | `'default'`
-| resourceId | n | The resource ID of the AKS cluster to render application resources into. Only required for Azure environments | `aksCluster.id`
-| identity | n | The cluster identity configuration | [See below](#identity) |
+| kind | y | The kind of container runtime to use, with `'kubernetes'` and `'aci'` currently supported. | `'kubernetes'`
+| namespace | n | The Kubernetes namespace to render application resources into, only required for Kubernetes environments. | `'default'`
+| resourceId | n | The resource ID of the AKS cluster to render application resources into, only required for Azure environments. | `aksCluster.id`
+| resourceGroup | n | The resource group to render application resources into, only required for ACI environments. | `'/subscriptions/mySubscriptionId/resourceGroups/my-resource-group'`
+| identity | n | The cluster identity configuration. | [See below](#identity) |
 
 ### identity
 
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
-| kind | y | The kind of identity. 'azure.com.workload' is currently only supported. | `'azure.com.workload'` |
-| oidcIssuer | n | The [OIDC issuer URL](https://azure.github.io/azure-workload-identity/docs/installation/self-managed-clusters/oidc-issuer.html) for your Kubernetes cluster. | `'{IssuerURL}/.well-known/openid-configuration'` |
+| kind | y | The kind of identity, `'azure.com.workload'`, `'userAssigned'`, `'systemAssigned'`, and `'systemAssignedUserAssigned'` are currently supported and defaults to `'systemAssigned'` if none provided. | `'systemAssigned'` |
```

Comment:
Let's clarify that system assigned is the default if your compute is ACI. nothing changes if your compute is Kubernetes

Reply:
done